cli/listings.sh

```diff
@@ -101,11 +100,8 @@ check() {
 
 report() {
     matchGAV $1
-    echo `formatGAVjson`
     tmpfile=`mktemp`
-    echo "[" > $tmpfile # WA until there is pretty print for reports
```

Comment:
`pretty-lookup.py` can now parse JSON objects on top of JSON list, so this is no longer required.